web/static/js/services/retro_channel.js

```diff
@@ -11,27 +11,42 @@ class RetroChannel {
   }
 }
 
+export const handlePresence = (presence, setPresences) => {
```

Comment:
are we introducing this helper due to ease of testing? it almost feels like the indirection of passing the `presence` and `setPresences` action makes this a little less grokkable? 

what if we kept it inline down below?
```es6
const presence = new Presence(retroChannel)

presence.onSync(() => {
    const users = presence.list((_token, presence) => (presence.user))
    setPresences(users)
  })


```

Reply:
Yeah - originally it was more complicated, but now that it's a two liner, makes sense to get rid of it (I also thought I would test it separately, but that was also unneeded).